packages/headless/src/integration-tests/search-app.test.ts

```diff
@@ -27,9 +27,7 @@ import {
   buildRelevanceSortCriterion,
```

Comment:
I was cleaning up the commerce tests, so I cleaned up the search ones as well. Instead of running in 20s, they run in 7ms now.